features/step_definitions/operator.rb

```diff
@@ -9,3 +9,125 @@
+        | f | #{operator_group_yaml} |
+        | p | NAME=etcd-og         |
+        | p | NAMESPACE=#{proj_name} |
+      })
+      raise "Error creating OperatorGroup: etcd-og" unless @result[:success]
+    end
+    logger.info("### operator group: etcd-og is installed successfully in #{proj_name} namespace")
+
+    unless subscription("#{name}").exists?
+      # Subscribe etcd operator
+      sub_etcd_yaml ||= "https://raw.githubusercontent.com/openshift-qe/v3-testfiles/master/olm/etcd-subscription-template.yaml"
+      step %Q/I process and create:/, table(%{
+        | f | #{sub_etcd_yaml}                      |
+        | p | NAME=#{name}                          |
+        | p | NAMESPACE=#{proj_name}                |
+        | p | SOURCENAME=community-operators        |
+        | p | SOURCENAMESPACE=openshift-marketplace |     
+      })
+      raise "Error creating subscription: #{name}" unless @result[:success]
+    end
+
+    step %Q/I wait for the "etcd-operator" deployment to appear/
+    step %Q/a pod becomes ready with labels:/, table(%{
+      | name=etcd-operator-alm-owned |
+    })
+    raise "Error creating #{name} pods in #{proj_name} project" unless @result[:success]
+    
+    step %Q/I run the :get client command with:/, table(%{
+      | resource | etcdcluster |                                                  
+    })
+    step %Q/the output should contain "No resources found"/
+    raise "Error find the etcdCluster resource in #{proj_name} project" unless @result[:success]
+    logger.info("### etcd operator: #{name} is installed successfully in #{proj_name} namespace")
+end
+
+Given /^etcdCluster "([^"]*)" is installed successfully in "([^"]*)" project$/ do | name, proj_name|
+  step %Q/I use the "#{proj_name}" project/
+  etcdCluster_yaml ||= "https://raw.githubusercontent.com/openshift-qe/v3-testfiles/master/olm/etcd-cluster-template.yaml"
+  step %Q/I process and create:/, table(%{
+    | f | #{etcdCluster_yaml} |
+    | p | NAME=#{name}         |
+    | p | NAMESPACE=#{proj_name} |
+  })
+  raise "Error creating etcdCluster: #{name} in project: #{proj_name}" unless @result[:success]
+  logger.info("### etcdCluster: #{name} is installed successfully in #{proj_name} namespace")
+end
+
+Given /^etcdCluster "([^"]*)" is removed successfully from "([^"]*)" project$/ do | name, proj_name|
+  step %Q/I run the :delete client command with:/, table(%{
+    | object_type       | etcdcluster  | 
+    | object_name_or_id | #{name}      |
+    | n                 | #{proj_name} |      
+  })
+  step %Q/the output should contain "deleted"/
+  step %Q/I wait for the resource "etcdcluster" named "#{name}" to disappear within 180 seconds/
+  step %Q/I ensure "#{name}" etcdcluster is deleted from the "#{proj_name}" project after scenario/
+  step %Q/I run the :get client command with:/, table(%{
+    | resource      | etcdcluster  | 
+    | resource_name | #{name}      |
+    | n             | #{proj_name} |      
+  })
```

Comment:
this step not needed, the `ensure deleted` step above already checks that

Reply:
Ok, I see, thanks!